src/main/java/org/osiam/Osiam.java

```diff
@@ -51,22 +51,20 @@
 public class Osiam extends SpringBootServletInitializer {
 
     private static final Map<String, Object> DEFAULT_PROPERTIES = ImmutableMap.<String, Object>of(
-            "osiam.home", System.getenv("HOME") + "/.osiam",
-            "spring.config.location", "classpath:/home/config/osiam.yaml,file:${osiam.home}/config/osiam.yaml"
+            "spring.config.location", "file:${osiam.home}/config/osiam.yaml"
     );
 
     public static void main(String[] args) {
         SpringApplication application = new SpringApplication();
-        application.setDefaultProperties(DEFAULT_PROPERTIES);
-
         String command = extractCommand(args);
         if ("initHome".equals(command)) {
```

Comment:
But seriously, I think it's ok to use it. I know, that, when reversed, it might be a little bit easier to comprehend. But OTOH it's a well-known idiom in the Java community. So I think there are not really arguments against nor for it.